### ISSUE TITLE
geth dev mode

### ISSUE THREAD

**Issue by Pman-me** [ethereum/go-ethereum#29538]
I upgraded the geth dev mode
and in a project for testing  many transactions are made in parallel
before upgrading, there was no problem in version 1.13.5
but in version 1.13.14, there is a problem : replacement transaction underpriced and also block number is always 0

what's wrong

**Comment by s1na** [comment]
Duplicate of https://github.com/ethereum/go-ethereum/issues/29475.
